### ISSUE TITLE
Recommend using statically linked binary

### ISSUE THREAD

**Issue by RamXX** [vmware/photon-controller#51]
I'm attempting to install the `photon` CLI only in an Alpine Linux Docker container to use in Concourse pipelines (ideal due to its tiny size). 

Getting the binary from here `wget https://github.com/vmware/photon-controller/releases/download/v1.0.0/photon-linux64` and running it produces `bash: ./photon-linux64: No such file or directory`, which is classic for a dynamically linked binary.

I've been unable to build from source on Alpine 3.4 so far, so I'd like to recommend to ship the binaries in the future statically linked, so any Linux distro can be used for this purpose.

**Comment by schadr** [comment]
just to make sure the execution permissions are set binary?

**Comment by RamXX** [comment]
Yes. I thought it was obvious, reason why I didn't specify it above. Thanks.

**Comment by schadr** [comment]
is there a way for you to share your dockerfile

**Comment by RamXX** [comment]
It's in my repo [here](https://github.com/RamXX/cfjump-alpine).
